flume/conf/README.md

```diff
@@ -0,0 +1,54 @@
+# Cygnus configuration Procedure
+
+From version 0.6.0 Cygnus can start multiple instances by renaming and configuring some of 
+templates files of this directory.
+
+There are two types of configuration template files:
+1- Files which is only one for all intances
+2- Files wich there are one per each intance that is wanted to run.
```

Comment:
Fix in [e46fe68](https://github.com/telefonicaid/fiware-connectors/commit/e46fe6894611cc927e1916e003da3d9a9b39a8ef)